person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand 1..1000
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services
+    @age >= 18 || @parent_permission == true
+  end
+
+  def of_age?
+    @age >= 18
+  end
+  private :is_of_age
+end
```

Comment:
- [ ] In line 18, `is_of_age` is not defined anywhere. Unless this is supposed to be `of_age` as it's seen in line 15. Make this change to clearly define the `of_age` method as private.

Reply:
Hi @mke2111, Thanks for spotting the error. I have adjusted the code